projects/dsp-ui/src/lib/viewer/directives/drag-drop.directive.ts

```diff
@@ -0,0 +1,34 @@
+import { Directive, EventEmitter, HostBinding, HostListener, Output } from '@angular/core';
+
+@Directive({
+    selector: '[dspDragDrop]'
+})
+export class DragDropDirective {
+
+    @HostBinding('style.background-color') background = '#f2f2f2';
+
+    @Output() fileDropped = new EventEmitter<any>();
```

Comment:
Could you use the actual type `FilesList` instead of `any`?

Reply:
if you want to test the directives `@Output`, this is a similar case with a test host component listening to the event:

https://github.com/dasch-swiss/dsp-ui-lib/blob/15aae792864228a85384a96b417934638cb4063b/projects/dsp-ui/src/lib/viewer/directives/text-value-html-link.directive.spec.ts#L10-L32

Reply:
Seems testing is "die unendliche Geschichte" 😃